email/1.3/modules/ROOT/edits/email-list.adoc

```diff
@@ -0,0 +1,210 @@
+= List Emails with Email Connector Examples - Mule 4
+:page-aliases: connectors::email/email-list.adoc
+
+Anypoint for Email (Email Connector) provides the *List - IMAP* and *List - POP3* operations to list emails from any part of the flow. The following examples illustrate how to configure these operations for the connector:
```

Comment:
```suggestion
Anypoint Connector for Email (Email Connector) provides the List - IMAP and List - POP3 operations, which list emails from any part of the flow. The following examples illustrate how to configure these operations for the connector:
```
Operation names are init. capped but not boldfaced unless you are referring to them as UI labels. GLOBAL